src/components/timers/Stopwatch.tsx

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react';
+import { Timer, TimerTitle } from '../../utils/Styled.tsx';
+import { milisecondsToTime } from '../../utils/helpers';
+import { TimersContext } from '../context/TimersContextProvider.tsx';
+import { RemoveButton, StatusDisplay, TimeDisplay } from '../generic/TimerComps.tsx';
+
+type StopwatchProps = {
+    id: string;
+};
+
+const Stopwatch = ({ id }: StopwatchProps) => {
+    const timersContext = useContext(TimersContext);
+    const t = timersContext.timers.find(timer => timer.id === id);
```

Comment:
this is repeated everywhere:
`const timersContext = useContext(TimersContext);
    const t = timersContext.timers.find(timer => timer.id === id);`

add a function to your context, for example `getTimerById` so you can DRY this